permissions/plugin-portshift-scanner.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "portshift-scanner"
+github: "jenkinsci/portshift-ci-plugin"
+paths:
+- "org/jenkins-ci/plugins/portshift-scanner"
+developers:
+- "portshift"
+security:
+  contacts:
+    jira: portshift
```

Comment:
You should not have a security section if the contact is the same as any of the developers in the "developers" section above.
```suggestion

```

Reply:
@slide , removed. please re review